nano/node/socket.hpp

```diff
@@ -124,5 +124,7 @@ class server_socket final : public socket
 	boost::asio::ip::tcp::endpoint local;
 	size_t max_inbound_connections;
 	void evict_dead_connections ();
```

Comment:
Possible improvement: These don’t have to be part of the class. Can be free functions in the .cpp file.